### PR TITLE
add initial speakers page

### PR DESCRIPTION
**Pull request by khjrtbrg**
Responding to issue https://github.com/OSFeels/OSFeels.com/issues/4

### CODE REVIEW

File: app/views/speakers.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      <h3 class="speakers--title <%= color_classes[color_index] %>"><%= peep.fetch :title%></h3>
+      <p class="speakers--abstract"><%= peep.fetch :abstract %></p>
+    </div>
+    <% color_index += (color_index == 2 ? -2 : 1) %>
+    <% row_index += 1 %>
+  <% if row_index % 2 == 0 %></div><% end %>
+<% end %>
+
+<%# Day 2 Speakers %>
+<%# Saving this for when the schedule is more firm. -jnf
```
Comment (khjrtbrg) [inline app/views/speakers.erb]: @jnf :+1: Good idea!

**Comment by jnf** [comment]
This code is so gross. :) I miss writing bunches of view code.

File: app/views/speakers.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+</div>
+
+<% 
+  color_classes = %w(blue pink purple)
+  color_index = 0
+  row_index = 0
+%>
+
+<% PRESENTERS.each do |peep| %>
+  <% if row_index % 2 == 0 %><div class="row speakers"><% end %>
```
Comment (strand) [inline app/views/speakers.erb]: Can we use `each_with_index` here?

**Comment by khjrtbrg** [comment]
@jnf What do you think about putting speakers' Twitter handles, websites, etc. next to their pictures and their bios at the bottom?

**Pull request merged**